man/docker-network-create.1.md

```diff
@@ -0,0 +1,55 @@
+% DOCKER(1) Docker User Manuals
+% Docker Community
+% OCT 2015
+# NAME
+docker-network-create - create a new network
+
+# SYNOPSIS
+**docker network create**
+
+[**-d**|**--driver**=[**name**]]     
+[**--help**]
+
+# DESCRIPTION
+
+Creates a new network. Driver accepts any of the following `name` values:
+
+
+| Value     | Description                                                                                                                                                                         |
+|-----------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `bridge`  | Uses Linux bridging and iptables to connect containers. This creates a single bridge, called `docker0` by default, and attaches a veth pair between the bridge and every container. |
+| `null`    | A noop implementation of the driver API, used only in cases where no networking is desired. This is to provide backward compatibility to the Docker's --net=none option.            |
+| `overlay` | Implements networking that can span multiple hosts using overlay network encapsulations such as VXLAN.                                                                              |
+| `remote`  | Not a driver but supports drivers over a remote transport. This allows you to [write a driver in the language of your choice](https://github.com/docker/libnetwork/blob/master/docs/remote.md).                                                       |
```

Comment:
I think we can remove mention of "remote" as it's no longer a user facing option. All drivers use the plugin discovery mechanism documented [here](https://docs.docker.com/extend/plugin_api/#plugin-discovery)